designs/index.md

```diff
@@ -7,25 +7,248 @@ title: Design Documents
+the PR only adds a link.
+
+When possible, the author [chooses a lead reviewer](#how-to-choose-a-lead-reviewer).
+Other reviewers are cc'ed. If the author didn't choose a lead reviewer, the
+Bazel sheriff will assign one, like for any other PR.
+
+Once the PR is sent, the reviewers can make some preliminary comments during the
+code review. For example, the lead reviewer can suggest extra reviewers, or
+point out missing information. The lead reviewer approves the PR when they
+believe the review process can start. It doesn't mean the proposal is perfect or
+will be approved; it means that the proposal contains enough information to
+start the discussion.
+
+
+### Announce the new proposal
+
+Once the PR is submitted, the author sends an announcement to [bazel-dev](https://groups.google.com/forum/#!forum/bazel-dev).
+
+Other groups may be cc'ed (e.g. [bazel-discuss](https://groups.google.com/forum/#!forum/bazel-discuss),
+to get feedback from Bazel end-users).
+
+
+### Iterate with reviewers
+
+Anyone interested can chime in and comment on the proposal. The author should
+try to answer questions, clarify the proposal, and address the concerns.
+
+Discussion should happen on the announcement thread. If the document is a Google
+Doc, comments may be used instead (but note that anonymous comments are
+allowed).
+
+
+### Update the status
+
+When the author believes the iteration round is complete, they send a new PR to
+update the status. The PR must be sent to the lead reviewer and cc the other
+reviewers.
+
+The lead reviewer approves the PR to officially accept or reject the proposal.
+It is the lead reviewer's responsibility to ensure that other reviewers agree
+with the decision.
+
+There must be at least 1 week between the first announcement and the approval of
+a proposal. This ensures that users had enough time to read the document and
+share their concerns.
+
+
+## Should I use markdown or Google Docs?
+
+Both are accepted. The author can decide what works best for them.
+
+Google Docs can be more effective for brainstorming, collaborative editing, and
+quick iteration. Suggested edits are also very valuable.
+
+Markdown files have some other benefits, including:
+
+*   Clean URLs for linking.
+*   Explicit record of revisions.
+*   No forgetting to set up access rights before publicizing a link.
+*   More easily searchable with search engines.
+*   Future-proofness: plain text is not at the mercy of any specific tool,
+    doesn't require an Internet connection.
+*   It is possible to update them even if the author is not around anymore.
+*   They can be processed automatically (e.g. update/detect dead links, fetch
+    list of authors, etc.).
+
+It is also possible to first iterate on a Google Doc, and then convert it to
+markdown for posterity.
+
+
+## Using Google Docs
+
+To make a [Google Doc](https://docs.google.com) world-readable, create it with a
+personal account. To share it, click on "Share", "Advanced", then "Change…", and
```

Comment:
I have trouble with 'personal'. It leaks personal identity into a business setting. Most of the people working on Bazel do it as part of their jobs, not as a personal quest.
Is the intent to mean "if your company uses gSuite, then create a doc with another identity, so that the doc can be shared outside that gSuite domain"?

I suspect that is the motivation, since most admins look down their account to prevent sharing outside it.

Reply:
> I suspect that is the motivation, since most admins look down their account to prevent sharing outside it.

Correct, like how we use `@bazel.build` accounts.